src/qt/guiutil.h

```diff
@@ -123,6 +123,13 @@ namespace GUIUtil
      */
     QString getDefaultDataDirectory();
 
+
+    /** Extract first suffix from filter pattern "Description (*.foo)" or "Description (*.foo *.bar ...).
+
+      @param[in] filter Filter specification such as "Comma Separated Files (*.csv)"
+    */
```

Comment:
Please consult with [Coding Style (Doxygen-compatible comments)](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments).

Reply:
Done in https://github.com/bitcoin-core/gui/pull/620/commits/c378535e28e0c54353fde9c4a66cb3134a3ee742.